test/unit/job_template_importer_test.rb

```diff
@@ -10,29 +10,26 @@ class JobTemplateImporterTest < ActiveSupport::TestCase
 
     let(:result) do
       name = 'Community Service Restart'
+      metadata = {
+        'model' => 'JobTemplate',
+        'kind' => 'job_template',
+        'name' => 'Service Restart',
+        'job_category' => 'Service Restart',
+        'provider_type' => 'SSH',
+        'feature' => remote_execution_feature.label,
+        'template_inputs' => [
+          { 'name' => 'service_name', 'input_type' => 'user', 'required' => true },
+          { 'name' => 'verbose', 'input_type' => 'user' }
+        ]
+      }
       text = <<-END_TEMPLATE
 <%#
-model: JobTemplateImporter
```

Comment:
I think `JobTemplateImporter` was chosen as a model just because `JobTemplate` already had `::import!` defined (which foreman-templates uses). This PR renames the import related methods a bit, this is described in the PR description.